src/handler/active_requests.rs

```diff
@@ -2,7 +2,7 @@ use super::*;
 use delay_map::HashMapDelay;
 use more_asserts::debug_unreachable;
 
-pub(crate) struct ActiveRequests {
+pub(super) struct ActiveRequests {
```

Comment:
this was required to avoid leaking the  type that models a request private to the handler